docs/contributor_guide/development_process.rst

```diff
@@ -7,6 +7,20 @@ Contributors should use their own forks of the repository. In their fork, they
 create feature branches off of :code:`master`, and their pull requests should
 target the :code:`master` branch. Maintainers are responsible for prompt
 review of pull requests.
+The `CODEOWNERS.md <https://github.com/fairlearn/fairlearn/blob/master/CODEOWNERS.md>`_
+file lists the subject matter experts for all parts of the repository
+including the documentation and examples.
+Every change requires approval from at least one of the listed subject matter
+experts, and a total of two approvals from current maintainers.
```

Comment:
I actually started wondering about this with my own PRs: do I need both of you to approve given that we only have three maintainers? Or do I count as one myself? Perhaps we need to loosen this a little bit in some way.